src/field_theory/adjoin.lean

```diff
@@ -468,37 +478,62 @@ begin
   { exact aeval_gen_minpoly F α }
 end
 
+section power_basis
```

Comment:
This section is the same as the `power_basis` section below, just moved up to just before where I needed it.